### PR TITLE
Fixed the interacting with cluster section for the ubuntu installation

### PR DESCRIPTION
**Pull request by CalvinHartwell**
I've modified some outdated commands in the master branch. I assume I may need to make this change in other branches as well, please let me know. The current steps don't work. 

Cheers,

- Calvin

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6905)

### CODE REVIEW

**Comment by CalvinHartwell** [comment]
Looks good to me, not sure why travis is failing: [https://deploy-preview-6905--kubernetes-io-master-staging.netlify.com/docs/getting-started-guides/ubuntu/installation/#interacting-with-the-cluster](https://deploy-preview-6905--kubernetes-io-master-staging.netlify.com/docs/getting-started-guides/ubuntu/installation/#interacting-with-the-cluster )

File: docs/getting-started-guides/ubuntu/installation.md [hunk 1]
```diff
@@ -172,16 +172,15 @@ Copy the kubeconfig file to the default location.
...
-Fetch a binary for the architecture you have deployed. If your client is a
-different architecture you will need to get the appropriate `kubectl` binary
-through other means. In this example we copy kubectl to `~/bin` for convenience,
-by default this should be in your $PATH.
+The next step is to install the kubectl client on your local machine. The recommended way to do this on Ubuntu is using the kubectl snap ([https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-with-snap-on-ubuntu](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-with-snap-on-ubuntu)):
 
 ```
-mkdir -p ~/bin
-juju scp kubernetes-master/0:kubectl ~/bin/kubectl
+# Run this on machine you wish to use to control the kubernetes cluster.
```
Comment (zacharysarah) [inline docs/getting-started-guides/ubuntu/installation.md]: Replace with:
```
# Run this on the machine you want to use to control the kubernetes cluster.
```

Comment (CalvinHartwell) [reply docs/getting-started-guides/ubuntu/installation.md]: @zacharysarah I'm confused, what exactly do you want me to change to fix this?

Comment (CalvinHartwell) [reply docs/getting-started-guides/ubuntu/installation.md]: @zacharysarah sorry to bump, but what exactly do you want me to change here?

Comment (zacharysarah) [reply docs/getting-started-guides/ubuntu/installation.md]: @calvinhartwell There’s a missing article. If you replace the entire line with the line I provided, that should take care of it.

**Comment by CalvinHartwell** [comment]
@zacharysarah is this change any better?

**Comment by zacharysarah** [comment]
@CalvinHartwell Yes, thanks! 

/approve

**Comment by zacharysarah** [comment]
Manually squashing and merging due to PR age.

**Pull request merged**